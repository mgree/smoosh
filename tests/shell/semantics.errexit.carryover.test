set -e
putsn() { echo "$@"; }
false && true
putsn "It should be executed"
false && true
/bin/echo hello
