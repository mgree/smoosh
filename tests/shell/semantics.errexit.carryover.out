It should be executed
hello
